test/MochaSpecRunner.html

```diff
@@ -0,0 +1,51 @@
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>Mocha Tests</title>
+    <script type="text/javascript" src="lib/webcomponents.js"></script>
+    <link href="lib/mocha.css" rel="stylesheet" />
+    <link rel="import" href="../lab/polymer/components/my-menu-button.html">
+</head>
+
+<body>
+    <my-menu-button style="position: absolute; top: 0; right: 0" href="subpage.html">Distributed <strong>Content</strong></my-menu-button>
```

Comment:
@warpech as for tests still passing, check this comment https://github.com/Palindrom/Palindrom/blob/fce35af45e5edbde5cb4ba4724ec5ad17c49b27f/test/specs/dom/index.js#L141